app/test_main.py

```diff
@@ -1,3 +1,20 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize("cat_years, dog_years, expected", [
+    (0, 0, [0, 0]),
+    (13, 13, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (20, 20, [1, 1]),
+    (23, 23, [1, 1]),
+    (27, 26, [2, 2]),
```

Comment:
The test case `(27, 26, [2, 2])` seems incorrect based on the task description. For 26 dog years, the expected human age should be 2, not 3. Please verify and correct this test case.